components/Navigation.js

```diff
@@ -2,30 +2,29 @@ import styled from "styled-components";
 import Image from "next/image";
 import homeIcon from "../public/images/homeIcon.png";
 import addIcon from "../public/images/addIcon.png";
-import { useRouter } from "next/router";
+import Link from "next/link";
 
 export default function Navigation() {
-  const router = useRouter();
   return (
     <StyledNavigation>
       <UlStyledList>
         <StyledList>
```

Comment:
I think you should take another look at what you call your styled components
Suggestion:
**You don't need `Ul` here**
`UlStyledList`: `List` or `StyledList`
**That's an Entry, not a List**
`StyledList`: `ListEntry` or `StyledListEntry`